descriptors/ModuleDescriptor-template.json

```diff
@@ -154,7 +149,11 @@
   ],
   "requires": [
     {
-      "id": "mod-fqm-manager",
+      "id": "query",
```

Comment:
`query` is a tad generic/broad.  Consider having mod-fqm-manager provider `fqm` or `fqm-query`.

Reply:
Good call! I went ahead and renamed it to `fqm-query`